pyre/scripts/cpyext-abi.py

```
@@ -0,0 +1,347 @@
+        "",
+        "# [declarations]",
+    ]
+    lines += [f"{n} :: {', '.join(a)} -> {r}" for n, (a, r) in sorted(declarations.items())]
+    lines += ["", "# [typedefs]"]
+    lines += [f"{n} :: {t}" for n, t in sorted(typedefs.items())]
+    RECORD.write_text("\n".join(lines) + "\n")
+    print(f"{len(declarations)} declarations, {len(typedefs)} typedefs -> {RECORD}")
+    return 0
+
+
+def command_check(args):
```

Ruff (0.16.1)

[warning] Unused function argument: `args`

(ARG001)